server/src/main/java/org/candlepin/controller/ContentAccessManager.java

```diff
@@ -509,7 +518,35 @@ private String createDN(Consumer consumer, Owner owner) {
     private byte[] createContentAccessDataPayload(Owner owner, Environment environment) throws IOException {
         // fake a product dto as a container for the org content
         Set<Product> containerSet = new HashSet<>();
-        CandlepinQuery<Product> ownerProduct = ownerProductCurator.getProductsByOwner(owner);
+        Set<String> productUuids = new HashSet<>();
+
+        Collection<String> activePoolIds = poolCurator.getActivePoolByOwnerId(owner.getId());
```

Comment:
This is an expensive way to fetch the product information for active pools. If the goal here is to fetch products for active pools, we should do exactly that: fetch active pools and then pull the products from them in code (functionally identical to what we're doing here), or write a new curator method that fetches only products from active entitlements of a given organization.

Reply:
Changed the logic to pull products & it's content directly in the code from active pools.

Not sure if we can use active entitlement to pull product's contents here, since active pool can exists without any active entitlement.